consensus/core/src/authority_node.rs

```diff
@@ -434,8 +435,7 @@ mod tests {
         async fn fetch_commits(
             &self,
             _peer: AuthorityIndex,
-            _start: Round,
-            _end: Round,
+            _commit_range: CommitRange,
```

Comment:
`start` and `end` are inclusive, which is different from CommitRange (inclusive..exclusive). Let's don't change them for this PR.